### ISSUE TITLE
About the dataset

### ISSUE THREAD

**Issue by abcdvzz** [ustc-vim/vegfru#2]
Veg-fru dataset has a test set which is much bigger than the train set and val set. I'd like to know if you can tell me which two of them are used as the training set and validation set during training phase. And which one of them is used in your paper to get the top-1 accuracy? Thank you very much.
